src/views/virtualmachines/details/tabs/configuration/details/components/DetailsSectionBoot.tsx

```diff
@@ -46,7 +52,7 @@ const DetailsSectionBoot: FC<DetailsSectionBootProps> = ({ canUpdateVM, vm, vmi
       <VirtualMachineDescriptionItem
         descriptionData={
           <div className={classNames({ 'text-muted': !canUpdateVM })}>
-            {getBootloaderTitleFromVM(vm)}
+            {getBootloaderTitleFromVM(instanceTypeVM)}
```

Comment:
Shouldn't we have something like `instanceTypeVM || vm` ? Is instanceTypeVM populated even when the VM do not have instanceType?

Reply:
u right i forgot